doc/man1/openssl-pkcs12.pod.in

```diff
@@ -85,8 +85,13 @@ The PKCS#12 export encryption and MAC options such as B<-certpbe> and B<-iter>
 and many further options such as B<-chain> are relevant only with B<-export>.
 Conversely, the options regarding encryption of private keys when outputting
 PKCS#12 input are relevant only when the B<-export> option is not given.
+
 The default encryption algorithm is AES-256-CBC with PBKDF2 for key derivation.
 
+When encountering problems loading legacy PKCS#12 files that involve,
+for example, C<RC2-40-CBC>,
```

Comment:
Not sure if `C<>` is right here

Reply:
Removed (in analogy to AES-256-CBC above).